packages/front-end/styles/_bootstrap-theme-overrides.scss

```diff
@@ -0,0 +1,372 @@
+// Global Bootstrap overrides that respect dark/light modes
```

Comment:
These are mostly Bootstrap but we have other components that are overridden here.